src/lib/calculate-polygon-center.js

```diff
@@ -0,0 +1,165 @@
+import { Client } from "pg";
+const client = new Client();
+client.connect();
+
+// client.query(`CREATE TABLE Polygons (
```

Comment:
Commented code?

Reply:
:fire:

Reply:
I left in the commented code to create the polygon table. It only needs to be ran once, hence the reason I commented it out. Although I figure it's worth keeping around